cmd/otel-agent/subcommands/status/command_test.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+package status
```

Comment:
Do you think it's possible to add a test that checks the output of the status command ? e.g. a string comparison. Would be a good check that this outputs what is expected.

Reply:
It's a bit difficult to add a unit test here due to the ddflareextension, but I just added some e2e tests to check the output of the status command

Reply:
Great, thanks for adding this